### ISSUE TITLE
Shift crafting only counting 1 crafted item, even if the output is 1>

### ISSUE THREAD

**Issue by DateInJapanese** [PikaMug/Quests#1235]
**What is your Quests version / build number (do _not_ say "latest")?**
v3.9.4-b110
This server is running CraftBukkit version git-Spigot-596221b-9a1fc1e (MC: 1.12) (Implementing API version 1.12-R0.1-SNAPSHOT)

**Describe the problem and how to replicate it. Any console errors to post _using Pastebin.com_?**
No console errors.

In my case I made a crafting quest for 20 cookies.
When crafting the cookies, shift clicking only gives progress for 1/20 Even though you get 8 cookies per craft.
When crafting the cookies and taking them without shift clicking on the cookies, it counts for 8/20.

Other plugins I have:
Itemizer
WE
FAWE
WG
VS
EssentialsX
ViaVersion
Vault
Citizens

**If applicable, can you provide an example quest from quests.yml _using Pastebin.com_?**
https://pastebin.com/VPXvZ042

**Comment by PikaMug** [comment]
Likely duplicate of #1067 

Works fine on my end. Are you able to replicate this with those plugins temporarily removed?

**Comment by DateInJapanese** [comment]
It's not the same as that, since that issue stated it doesn't count it at all. This one counts at least 1.
I'll have to test it later without the other plugins though, only thing that could mess with it I believe would be world guard. Other plugins have noting to do with it, but I'll test all.

**Comment by DateInJapanese** [comment]
With only the Quests plugin active, it still occurs.

Shift left click = 1 complete
Left click = 8 complete, 
for the cookie crafting that is.

**Comment by PikaMug** [comment]
So I don't have a 1.12 server to test on at the moment, but shift-clicking worked fine on a 1.9.4 server:

![cookies](https://user-images.githubusercontent.com/2267126/80181524-06638300-85d3-11ea-9db1-17d71bb37a16.png)

Installed plugins were:

BKCommonLib 1.15.2-v1
Citizens 2.0.26-b1804
Denizen 1.1.3-b1704
EssentialsX 2.17.0.0
MythicMobs 4.9.0
Quests 3.9.4-b114

**Comment by DateInJapanese** [comment]
Did you craft 8 stacks at once?
Try doing 1 craft each time, as that's what I meant.

So 8 cookies first, then check /quest
Both for shift clicks and non-shift clicks.

**Comment by PikaMug** [comment]
@DateInJapanese Fairly certain I've resolved the issue here, but please try the following development build and report back with results:
https://ci.codemc.io/job/PikaMug/job/Quests/127/

**Comment by DateInJapanese** [comment]
Works for cookies.
But not for a few other recipes I tested.

Cookies work fine, shift clicking single or multiple stacks at the time count for 8/craft, as it should.

=

Other recipes such as bone meal and blaze powder (and likely similar,) don't work.
Bones > 3 Bone Meal/craft
Blaze Rods > 2 Blaze powder/craft

Whether you're shift clicking, normal clicking, or multi crafting.
It always counts the bone meal for 3 crafts and blaze powder for 2 crafts, even if the final output is more.

Here is the quest I made, if you want to use it;
https://pastebin.com/08WYzD98
